docs/how-to/rest-api-and-dynamodb.md

```diff
@@ -0,0 +1,1364 @@
+```
+
+As you can see, `cURL` is pretty powerful and we’ll be using it again and again throughout this guide.
+
+### Using AWS DynamoDB to store & retrieve data
+
+![Databases.png](../../static/img/tutorial/rest-api/Databases.png)
+
+As we just said, RESTful APIs are built around databases. But *what* really is a database?
+
+Well, a database is a set of inter-connected collections of information. For example, a database may contain a collection of users and a collection of products. Users and products may be connected through shopping carts. (Product A exists in User B’s shopping cart)
+
+Some databases require every collection to have a fixed-set of data-points. For example, a user might have a name, a phone number and an email address. That would be the constraint in a **schema**-enabled database. Collections might also be seen as tables in this case, with a fixed number of columns that have their own data-type. (such as text, integer, booleans, and more)
+
+On the other hand, schema-less databases have no such limitation. Each item in a collection is a JSON document, and it can have any structure desired. Such databases are also referred to as noSQL databases, while their counterparts are known as SQL databases. A user, for example, might have the following entry in a noSQL database:
+
+```json
+{
+	"name": "John Doe",
+	"isActive": true,
+	"email": "example@domain.tld"
+}
+```
+
+SQL is a language used for processing schema-enabled databases. It allows for the efficient searching and processing of information even on the biggest databases out there.
+
+But with DynamoDB, a noSQL database engineered by Amazon’s Web Services (AWS), you still get to enjoy almost the whole power of SQL whilst feeling the freedom that comes with schema-less databases. It’s called [PartiQL](https://partiql.org/).
+
+We’re building a bikes shop API, so our database should contain a collection of bikes. It’s a mix of Strings, Numbers and whole Objects.
+
+Keep reading to see how we’ll be shaping our API around this database.
+
+## Getting started with a RESTful API
+
+**Requirements:**
+
+- Node (16+)
+- NPM (8+)
+- cURL (7+)
+
+and it won’t hurt to take a quick refresher on ES6 JavaScript programming.
+
+We just spent a lot of time trying to understand REST in theory, but nothing can beat learning by practice. So let’s create our own API. Our theme is a bikes shop, so that’s how we’ll be modelling our RESTful routes.
+
+Cyclic offers its users with a variety of starter templates, and REST APIs are included, of course. So let’s begin our exciting journey by visiting [Cyclic’s starters](https://app.cyclic.sh/#/deploy) and deploying the [Database API](https://app.cyclic.sh/#/deploy).
+
+This will fork the respective starter repository to your own Github account (that you signed in with).
+
+[![Deploy to Cyclic](https://deploy.cyclic.sh/button.svg)](https://app.cyclic.sh/#/deploy)
```

Comment:
I think this will try to fork the docs repo? I think you meant the following, just not sure which owner and repo:

```markdown
[![Deploy to Cyclic](https://deploy.cyclic.sh/button.svg)](https://deploy.cyclic.sh/:owner/:repo)
```